### ISSUE TITLE
Titanic Dataset - 使用 Pytorch 搭建神經網路 + 測試 overfitting | Shannon's Blog 🐟 金魚腦聖地

### ISSUE THREAD

**Issue by ShannonHung** [ShannonHung/ShannonHung.github.io#33]
https://shannonhung.github.io/posts/pytorch-titanic-nn.html 

 Reference  Ref:很詳細說明Titanic資料裡面的結構狀態  前言 最近選了一堂AI課程，有一個作業是我們寫出一個Nerual Network，並且使用Titanic Dataset來訓練，並且透過增加 hidden layer 跟 neurons 的方式實現overfitting，並透過 dropout 或其他方法來消除 overfitting 的影響。 在此紀錄ㄧ下作業撰寫的過程
